Tests/KlaviyoUITests/IAFNativeBridgeEventTests.swift

```diff
@@ -24,11 +24,23 @@ struct IAFNativeBridgeEventTests {
         }
         """
 
+        let deepLinkEvent = """
+        {
+            "ios": "klaviyotest://settings",
+            "android": "klaviyotest://settings"
+        }
+        """
+
         let data = json.data(using: .utf8)!
```

Comment:
```suggestion
        let data = try #require(json.data(using: .utf8))
```
this allows us to avoid force-unwrapping. It'll throw an error and the test will fail if the `json.data()` command fails